dev-embedded/esptool/esptool-4.9.1.ebuild

```diff
@@ -33,21 +33,14 @@ BDEPEND="
 		dev-python/wheel[${PYTHON_USEDEP}]
 	')
 	test? ( $(python_gen_cond_dep '
-		dev-python/cffi[${PYTHON_USEDEP}]
-		dev-python/requests[${PYTHON_USEDEP}]
```

Comment:
The commit message doesn't mention why you're changing these. Please include details.

I see that requests is only needed by a test we already deselect because it needs the network. How about the others?

Reply:
It's not just about testing w/o them, it's about grepping to see if they're still used, and identifying when they stopped being used (did upstream replace it with something else? is it really gone? or what?)